.github/workflows/on-pull-request-run-tests.yml

```diff
@@ -327,7 +330,7 @@ jobs:
   awx-e2e:
     name: AWX E2E
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 20
```

Comment:
we do not want to bump the timeout up

Reply:
It was just a test to verify that the changes were not the ones to cause test failures, I will update.